pyvista/core/filters/data_set.py

```diff
@@ -3156,6 +3158,10 @@ def sample(
         weighting for nearby points.  If there is cell topology, `sample` is
         usually preferred.
 
+        The point data 'vtkValidPointMask' stores whether the point could be sampled
+        with a value of 1 meaning successful sampling and value of 0 meaning
+        unsuccessful.
```

Comment:
Sorry for nitpick. But I want to split the sentence.
```suggestion
        The point data 'vtkValidPointMask' stores whether the point could be sampled
        with a value of 1 meaning successful sampling. And a value of 0 means
        unsuccessful.
```